src/llama_stack_api/registries.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+
+from datetime import datetime
+from enum import StrEnum
+from typing import Literal
+
+from pydantic import BaseModel, Field
+
+from llama_stack_api.resource import Resource, ResourceType
+from llama_stack_api.schema_utils import json_schema_type
+
+
+@json_schema_type
+class RegistryType(StrEnum):
+    """Type of registry."""
+
+    MCP = "mcp"
```

Comment:
what other registries would there be?

Reply:
There could be agent registries in the future once A2A is developed further